pkgs/by-name/li/libwacom/package.nix

```diff
@@ -0,0 +1,94 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  glib,
+  pkg-config,
+  udev,
+  libevdev,
+  libgudev,
+  python3,
+  valgrind,
+}:
+
+let
+  testsDisabled = "-Dtests=disabled";
+  testsEnabled = "-Dtests=enabled";
+  replaceDisabledWithEnabled =
+    flag:
+    lib.replaceStrings [
+      testsDisabled
+      testsEnabled
+    ] flag;
+in
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libwacom";
+  version = "2.12.2";
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "linuxwacom";
+    repo = "libwacom";
+    rev = "libwacom-${finalAttrs.version}";
+    hash = "sha256-dxnXh+O/8q8ShsPbpqvaBPNQR6lJBphBolYTmcJEF/0=";
+  };
+
+  postPatch = ''
+    patchShebangs test/check-files-in-git.sh
+  '';
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    ninja
+    python3
+  ];
+
+  buildInputs = [
+    glib
+    udev
+    libevdev
+    libgudev
+  ];
+
+  # Tests are in the `tests` pass-through derivation.
```

Comment:
```suggestion
  # Tests are in the `tests` pass-through derivation because one of them is flaky, frequently causing build failures.
```

Reply:
Will do. I'll also mention valgrind, which I suspect (due to personal experience) is the root cause of the slowness under load.